specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2017-11-03-preview/hanaonazure.json

```diff
@@ -220,6 +220,11 @@
                     "type": "string",
                     "description": "Resource location"
                 },
+                "powerState": {
```

Comment:
This cannot be a top level resource property. Must be a part of the `properties` propety

Reply:
Got it, updated

Reply:
Please check if this is how the service is consuming/returning the resource information

Reply:
I've just updated our API to move the power state. here is a sample:
```
{
    "id": "/subscriptions/c5c14913-17c5-4bb7-a8d4-5fb40d0a4823/resourceGroups/rg10004/providers/Microsoft.HanaOnAzure/hanaInstances/westusbmiName",
    "name": "westusbmiName",
    "type": "Microsoft.HanaOnAzure/hanaInstances",
    "location": "westus",
    "tags": {
     "owner": "alias4"
    },
    "properties": {
     "hanaInstanceId": "2e30e358-7ceb-42d0-b7f4-cb7e41e1a8bc",
     "powerState": "started",
     "hardwareProfile": {
      "hardwareType": "Cisco_UCS",
      "hanaInstanceSize": "S72"
     },
     "networkProfile": {
      "networkInterfaces": [
       {
        "ipAddress": "123.123.123.123"
       }
      ],
      "circuitId": "/subscriptions/c5c14913-17c5-4bb7-a8d4-5fb40d0a4823/resourceGroups/rg10004/providers/Microsoft.Network/expressRouteCircuits/westusbmiName"
     },
     "storageProfile": {
      "nfsIpAddress": "123.123.123.124"
     },
     "osProfile": {
      "computerName": "westusbmiName",
      "osType": "SLES 12 SP2",
      "version": "12 SP2"
     }
    }
   }
```

Reply:
Looks good.
As a sidenote, it looks like `powerstate` could be an enum, you could simply use an `x-ms-enum`

Reply:
`powerstate` is an enum now